api_tests/lib_sdk/actors/actor.ts

```diff
@@ -838,6 +838,21 @@ export class Actor {
             return await this.pollSwapDetails(swapUrl, iteration);
         }
     }
+
+    public async createLnInvoice(sats: number) {
```

Comment:
IMHO this is also bikeshedding because in the final version this will become useless/to be removed as we expect the SDK to do the creation of a hodl invoice. The e2e should not be calling this function.